administrator/components/com_media/libraries/media/file/adapter/local.php

```diff
@@ -277,20 +277,16 @@ private function getPathInformation($path)
 		$obj->modified_date           = $modifiedDate->format('c', true);
 		$obj->modified_date_formatted = (string) $modifiedDate; // TODO use format from config
 		$obj->mime_type               = mime_content_type($path);
+		$obj->width                   = 0;
+		$obj->height                  = 0;
 
-		try
+		if (strpos($obj->mime_type, 'image/') === 0 && in_array(strtolower($obj->extension), array('jpg', 'jpeg', 'png', 'gif', 'bmp')))
 		{
 			// Get the image properties
 			$props       = JImage::getImageFileProperties($path);
 			$obj->width  = $props->width;
 			$obj->height = $props->height;
 		}
-		catch (Exception $e)
```

Comment:
I'v removed that and opened issue #117 as we don't have a defined way of exception handling.